.jenkins_nightly

```diff
@@ -103,16 +103,19 @@ pipeline {
                           '''
                     }
                 }
-                stage('HIP-ROCM-6.0') {
+                stage('HIP-ROCM-6.1') {
                     agent {
                         dockerfile {
                             filename 'Dockerfile.hipcc'
                             dir 'scripts/docker'
-                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-20.04:6.0.2-complete'
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-20.04:6.1.1-complete'
                             label 'rocm-docker && AMD_Radeon_Instinct_MI210'
                             args '-v /tmp/ccache.kokkos:/tmp/ccache --device=/dev/kfd --device=/dev/dri --security-opt seccomp=unconfined --group-add video --env HIP_VISIBLE_DEVICES=$HIP_VISIBLE_DEVICES'
                         }
                     }
+                    environment {
+                        GTEST_FILTER = '-hip_hostpinned.view_allocation_large_rank'
+                    }
```

Comment:
Maybe to create an issue to reenable this in the future? I wonder what trigger would help us to remember to reenable this in the future.

Reply:
I can add a FIXME if you want.

Reply:
Adding a short FIXME comment stating why this is disabled would be useful.